rskj-ubuntu-installer/rskj_package_bionic/debian/changelog

```diff
@@ -1,3 +1,13 @@
+rskj (0.6.0~bionic) bionic; urgency=medium
```

Comment:
Shouldn't this be `critical`? without this upgrade the node will stop syncing once the network upgrade gets active

Reply:
We're having this in mind for future versions. This one was already released